wqflask/base/webqtlCaseData.py

```diff
@@ -66,7 +66,7 @@ def class_outlier(self):
 
     @property
     def display_value(self):
-        if self.value:
+        if self.value != None:
```

Comment:
Should we not use self.value is not None: Comparisons to singletons like None should always be done with
'is' or 'is not', never the equality operators.